python/agents/polymarket/polymarket.py

```
@@ -0,0 +1,481 @@
+# core polymarket api
+# https://github.com/Polymarket/py-clob-client/tree/main/examples
+
+import os
+import pdb
+import time
+import ast
+import requests
```

Ruff (0.11.9)

`requests` imported but unused

Remove unused import: `requests`

(F401)